template/androidApp/src/androidTest/java/io/bloco/template/ExampleInstrumentedTest.kt

```diff
@@ -15,10 +19,21 @@ import org.junit.Assert.*
  */
 @RunWith(AndroidJUnit4::class)
 class ExampleInstrumentedTest {
+
+    @get:Rule
+    val composeTestRule = createComposeRule()
+
+    init {
+        io.bloco.core.commons.RootLogger.setup({ _: Level, _: String, message: String, _: Throwable? ->
```

Comment:
Could we import `io.bloco.core.commons.RootLogger` instead?

Reply:
Also, ideally, by default the RootLogger comes with this logger, and it is swapped out if any other logger is added.